soccer/planning/EscapeObstaclesPathPlanner.cpp

```diff
@@ -53,20 +54,24 @@ Point EscapeObstaclesPathPlanner::findNonBlockedGoal(
     Point goal, boost::optional<Point> prevGoal, const ShapeSet& obstacles,
     int maxItr) {
     if (obstacles.hit(goal)) {
-        FixedStepTree goalTree;
-        goalTree.init(goal, &obstacles);
-        goalTree.step = stepSize();
+        auto stateSpace = make_shared<RoboCupStateSpace>(
```

Comment:
Yeah I mostly did it this way to avoid the complexity of handling field size changes.